internal/services/costmanagement/export_resource_group_resource_test.go

```diff
@@ -195,15 +199,17 @@ resource "azurerm_resource_group_cost_management_export" "import" {
   recurrence_type              = azurerm_resource_group_cost_management_export.test.recurrence_type
   recurrence_period_start_date = azurerm_resource_group_cost_management_export.test.recurrence_period_start_date
   recurrence_period_end_date   = azurerm_resource_group_cost_management_export.test.recurrence_period_start_date
+  partition_data_enabled               = azurerm_resource_group_cost_management_export.test.partition_data
```

Comment:
We only specify what was supplied in the configuration used to create the resource, since we're using the basic config which should not include `Optional` properties, this should be removed 
```suggestion
```